libs/causality/src/lib/CausalAnalysisDashboard/Controls/CausalAnalysisView/CausalIndividualView/LargeCausalIndividualView/LargeCausalIndividualChart.tsx

```diff
@@ -149,6 +151,10 @@ export class LargeCausalIndividualChart extends React.PureComponent<
   }
 
   private onRevertButtonClick = (): void => {
+    this.props.telemetryHook?.({
+      level: TelemetryLevels.Trace,
```

Comment:
Should this be ButtonClick instead of Trace?

Reply:
Absolutely!